collated/main/A0153440R.md

```diff
@@ -0,0 +1,806 @@
+# A0153440R
+###### \java\jym\manager\logic\commands\AddCommand.java
+``` java
+    public AddCommand(String description, Object ... objects) throws IllegalValueException{
+    	if(objects.length > 3){//f**k in this case - this should never happen b/c we control parser. max args is date(s), location, priority
+    		throw new IllegalArgumentException();
+    	}
+    	boolean isTask = true; //by default
+    	for(Object o : objects){
+    		if(o instanceof List){
+    			List<LocalDateTime> d = (List<LocalDateTime>)o;
+    			if(d.size() > 1)
+    				isTask = false;
+    			else
+    				o = d.get(0);
+    		}
+    	}
+//      final Set<Tag> tagSet = new HashSet<>();
```

Comment:
Remove dead code. Since we are using revision control, you can remove dead code without worrying about losing it.